libphal/phal_exception.H

```diff
@@ -26,6 +26,7 @@ enum ERR_TYPE {
 	SBE_FFDC_NO_DATA,
 	SBE_BOOT_SEEPROMS_BAD,
 	SBE_MEASUREMENT_SEEPROMS_BAD,
+	SBE_REIPL_WITHOUT_SWITCHING,
```

Comment:
**Commit message:**

>Adding support for recovery action RESTART_SBE/
RESTART_CBS.

Please update what needs to be done for the RESTART_SBE and RESTART_CBS recovery action in the commit message.